frontend/src/app/modules/common/path-helper/path-helper.service.ts

```diff
@@ -104,6 +104,10 @@ export class PathHelperService {
     }
   }
 
+  public attachmentContentPath(attachmentIdentifier:number|string) {
+    return this.staticBase + '/attachments/' + attachmentIdentifier + '/content';
+  }
+
   public ifcModelsPath(projectIdentifier:string) {
     return this.staticBase + `/projects/${projectIdentifier}/ifc_models`;
```

Comment:
`${this.staticBase}/projects/${projectIdentifier}/ifc_models`;
Be completely cool ;)